pypeit/core/datacube.py

```diff
@@ -941,8 +940,32 @@ def compute_weights_frompix(all_ra, all_dec, all_wave, all_sci, all_ivar, all_id
             Number of pixels used for determining smoothly varying S/N ratio
             weights.  This is currently not required, since a relative weighting
             scheme with a polynomial fit is used to calculate the S/N weights.
-        relative_weights (bool, optional):
-            Calculate weights by fitting to the ratio of spectra?
+        weight_method: (`str`, optional)
+           Weight method to be used in `coadd.sn_weights`. Default is 'auto'.
+           Options are 'auto', 'constant', 'relative', or 'ivar'. The defaulti is'auto'.
```

Comment:
typo: `The default is 'auto'`

Also, should the options listed also include `uniform` and `wave_dependent`?

Reply:
Fixed.